Makefile

```diff
@@ -105,36 +105,36 @@ SWARM_HOSTS = $(shell docker node ls --format="{{.Hostname}}" 2>$(if $(IS_WIN),N
 
 define _docker_compose_build
 export BUILD_TARGET=$(if $(findstring -devel,$@),development,production);\
-pushd services; docker buildx bake --file docker-compose-build.yml $(if $(target),$(target),); popd;
+pushd services; \
+docker buildx create --name osparc-builder --use || true; \
+docker buildx bake \
+	$(if $(findstring -devel,$@),,\
+	$(foreach service, $(SERVICES_LIST),\
+		--set $(service).cache-from="type=local,src=/tmp/.buildx-cache/$(service)" \
+		--set $(service).cache-to="type=local,mode=max,dest=/tmp/.buildx-cache/$(service)" \
+	)\
+	)\
+	--set *.output="type=docker,push=false" \
+	--file docker-compose-build.yml $(if $(target),$(target),); \
+popd; \
+docker buildx rm osparc-builder;
 endef
 
 rebuild: build-nc # alias
 build build-nc: .env ## Builds production images and tags them as 'local/{service-name}:production'. For single target e.g. 'make target=webserver build'
-ifeq ($(target),)
-	# Compiling front-end
-	$(MAKE_C) services/web/client compile
-
-	# Building services
+	# Building service(s) $(target)
 	$(_docker_compose_build)
-else
-ifeq ($(findstring static-webserver,$(target)),static-webserver)
-	# Compiling front-end
-	$(MAKE_C) services/web/client clean compile
-endif
-	# Building service $(target)
-	$(_docker_compose_build)
-endif
 
 
 build-devel build-devel-nc: .env ## Builds development images and tags them as 'local/{service-name}:development'. For single target e.g. 'make target=webserver build-devel'
 ifeq ($(target),)
 	# Building services
 	$(_docker_compose_build)
 else
-ifeq ($(findstring static-webserver,$(target)),static-webserver)
-	# Compiling front-end
-	$(MAKE_C) services/web/client touch compile-dev
-endif
+# ifeq ($(findstring static-webserver,$(target)),static-webserver)
+# 	# Compiling front-end
+# 	$(MAKE_C) services/static-webserver/client touch compile-dev
+# endif
```

Comment:
should this be removed?

Reply:
hmm well for now I don't have the devel mode covered.
Also once it's there I would like people to try this PR before I merge it since this will affect everyone local development